### ISSUE TITLE
Getting error when using "Merge from" command

### ISSUE THREAD

**Issue by sir-captainmorgan21** [shaharkazaz/vscode-git-merger#20]
![image](https://user-images.githubusercontent.com/7041924/38034542-85039294-3270-11e8-9652-d7dd97bc98d0.png)

I type the "Merge from" command and click enter. I get the above error.

**Comment by shaharkazaz** [comment]
Thanks for opening the issue.
Can you please elaborate?
What is you OS & VSCode version? 
When merging through git cli you have no errors?

**Comment by sir-captainmorgan21** [comment]
OS - macOS High Serria
VSCode Version - 1.21.1
Right I can run the command through command line
I think it is trying to run the command in the directory that contains all of my sub-directories that actually contain the repos. So when I "open" in VS Code I open my "Development" directory that had folders for each repo inside. When I run the git for-each-ref in the command line inside the "Development" directory, I get the same error your extension is getting.

**Comment by sir-captainmorgan21** [comment]
Here is a screenshot of my directories so you can get an idea.

![image](https://user-images.githubusercontent.com/7041924/38035906-71d66a04-3273-11e8-86f9-715125f2825e.png)

**Comment by shaharkazaz** [comment]
Thanks for the info ! Really gave me the idea for the problem.
I will look at it this weekend and try to find the best solution for it. 
Maybe a drop-down menu with the available repos. 
Thanks again for using my extension and helping me keep it up and running 👍

**Comment by sir-captainmorgan21** [comment]
Absolutely! A drop down for which repo would be nice (If you working directory has multiple sub directories with their respective repos). Would a work around for now be to open inside the specific directory I want to perform the merge in?

**Comment by sir-captainmorgan21** [comment]
I just tested and confirmed that is a good work around for now (If anyone else comes across this issue)
